WordPress/src/main/java/org/wordpress/android/viewmodel/history/HistoryViewModel.kt

```diff
@@ -103,6 +105,7 @@ class HistoryViewModel @Inject constructor(
             }
 
             items.add(item)
+            revisionsList.add(item)
```

Comment:
For [mRevisions.indexOf(mRevision)](https://github.com/wordpress-mobile/WordPress-Android/pull/8481/files#diff-4e1238a1f469995a58fd6d895e20640dR79) to work properly, we will also have to update revision in this list after we receive users avatar and display name in [fetchRevisionAuthorDetails](https://github.com/wordpress-mobile/WordPress-Android/pull/8481/files#diff-1b89f79e7b16cacd6c3ba7604c9b3ef8R124).

[This PR](https://github.com/wordpress-mobile/WordPress-Android/pull/8483) should fix similiar issue in `RevisionItemViewHolder` .

Reply:
Nice catch!  I added updating the revisions list after fetching the author details in d808325.